### PR TITLE
DOCS-1190: Document `obstacles_distance` segmenter

### PR DESCRIPTION
**Pull request by sguequierre**
* Add `"namespace"` for all segmenter models
* Document `obstacles_distance` segmenter
* Edit intro to segmenter page to point to nav service tutorial for example of working with a segmenter for obstacle avoidance (can remove if think not appropriate, not 100% sure if fits)

### CODE REVIEW

File: docs/services/vision/segmentation.md [hunk 1]
```diff
@@ -20,6 +24,7 @@ The types of segmenters supported are:
 - [**Obstacles point cloud (`obstacles_pointcloud`)**](#configure-an-obstacles_pointcloud-segmenter): A segmenter that identifies well-separated objects above a flat plane.
 - [**Object detector (`detector_3d_segmenter`)**](#configure-a-detector_3d_segmenter): This model takes 2D bounding boxes from an object detector and projects the pixels in the bounding box to points in 3D space.
 - [**Obstacles depth (`obstacles_depth`)**](#configure-an-obstacles_depth-segmenter): A segmenter for depth cameras that returns the perceived obstacles as a set of 3-dimensional bounding boxes, each with a Pose as a vector.
+- [**Obstacles distance (`obstacles_distance`)**](#configure-an-obstacles-distance-segmenter): A segmenter that takes point clouds from a camera input and returns the average single closest point to the camera as a perceived obstacle.
```
Comment (sguequierre) [inline docs/services/vision/segmentation.md:27]:
```suggestion
- [**Obstacles distance (`obstacles_distance`)**](#configure-an-obstacles distance-segmenter): A segmenter that takes point clouds from a camera input and returns the average single closest point to the camera as a perceived obstacle.
```

File: docs/services/vision/segmentation.md [hunk 2]
```diff
@@ -20,6 +24,7 @@ The types of segmenters supported are:
 - [**Obstacles point cloud (`obstacles_pointcloud`)**](#configure-an-obstacles_pointcloud-segmenter): A segmenter that identifies well-separated objects above a flat plane.
 - [**Object detector (`detector_3d_segmenter`)**](#configure-a-detector_3d_segmenter): This model takes 2D bounding boxes from an object detector and projects the pixels in the bounding box to points in 3D space.
 - [**Obstacles depth (`obstacles_depth`)**](#configure-an-obstacles_depth-segmenter): A segmenter for depth cameras that returns the perceived obstacles as a set of 3-dimensional bounding boxes, each with a Pose as a vector.
+- [**Obstacles distance (`obstacles_distance`)**](#configure-an-obstacles distance-segmenter): A segmenter that takes point clouds from a camera input and returns the average single closest point to the camera as a perceived obstacle.
```
Comment (sguequierre) [inline docs/services/vision/segmentation.md:27]:
```suggestion
- [**Obstacles distance (`obstacles_distance`)**](#configure-an-obstacles_distance-segmenter): A segmenter that takes point clouds from a camera input and returns the average single closest point to the camera as a perceived obstacle.
```

File: docs/services/vision/segmentation.md [hunk 3]
```diff
@@ -62,6 +67,7 @@ Add the vision service object to the services array in your raw JSON configurati
     {
     "name": "<segmenter_name>",
     "type": "vision",
+    "namespace": "rdk",
```
Comment (andf-viam) [inline docs/services/vision/segmentation.md:70]: Thanks for these!

File: docs/services/vision/segmentation.md [hunk 4]
```diff
@@ -325,6 +336,76 @@ If you choose not to configure the frame system, you can still identify single p
...
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+Navigate to your robot's **Config** tab on the [Viam app](https://app.viam.com/robots).
+Click the **Services** subtab and click **Create service** in the lower-left corner.
+Select the `Vision` type, then select the `Obstacles Distance` model.
+Enter a name for your service and click **Create**.
+
+In your vision service's panel, fill in the **Attributes** field.
```
Comment (andf-viam) [inline docs/services/vision/segmentation.md:354]:
```suggestion
In your vision service's configuration panel, fill in the **Attributes** field with the following:
```

File: docs/services/vision/segmentation.md [hunk 5]
```diff
@@ -325,6 +336,76 @@ If you choose not to configure the frame system, you can still identify single p
...
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+    {
+        "name": "my_segmenter",
+        "type": "vision",
+        "namespace": "rdk",
+        "model": "obstacles_distance"
```
Comment (andf-viam) [inline docs/services/vision/segmentation.md:391]:
```suggestion
        "model": "obstacles_distance",
```

File: docs/services/vision/segmentation.md [hunk 6]
```diff
@@ -325,6 +336,76 @@ If you choose not to configure the frame system, you can still identify single p
...
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following parameters are available for a `obstacles_distance` segmenter:
+
+<!-- prettier-ignore -->
+| Parameter | Inclusion | Description |
+| --------- | --------- | ----------- |
+| `num_queries`| Optional  | How many times the model should call [`GetPointCloud()`](/components/camera/#getpointcloud) before taking the average of the measurements and returning the single closest point. An integer between `1` and `20`. <br> Default: `10`  |
```
Comment (andf-viam) [inline docs/services/vision/segmentation.md:407]:
```suggestion
| `num_queries`| Optional  | How many times the model should call [`GetPointCloud()`](/components/camera/#getpointcloud) before taking the average of the measurements and returning the single closest point. Accepts an integer between `1` and `20`. <br> Default: `10`  |
```

**Review by andf-viam — APPROVED** [review]
LGTM, just some tiny suggestions. Thanks!!

File: docs/services/vision/segmentation.md (hunk 5 above)
Comment (sguequierre) [reply docs/services/vision/segmentation.md:391]: Done, TY

File: docs/services/vision/segmentation.md (hunk 6 above)
Comment (sguequierre) [reply docs/services/vision/segmentation.md:407]: Done!

**Review by bhaney — APPROVED** [review]

File: docs/services/vision/segmentation.md [hunk 7]
```diff
@@ -11,7 +11,11 @@ tags: ["vision", "computer vision", "CV", "services", "segmentation"]
 _Changed in [RDK v0.2.36 and API v0.1.118](/appendix/release-notes/#25-april-2023)_
 
 _3D Object Segmentation_ is the process of separating and returning a list of the identified "objects" from a 3D scene.
-The "objects" are a list of point clouds with associated metadata, like the label, the 3D bounding box, and center coordinates of the object.
+The "objects" are usually a list of point clouds with associated metadata, like the label, the 3D bounding box, and center coordinates of the object.
+They can also be single points, like with the `obstacles_distance` model.
+
+3D object segmentation is useful for obstacle detection.
+See our guide [Navigate with a Rover Base](/tutorials/services/navigate-with-rover-base/) for an example of automating obstacle avoidance with 3D object segmentation.
```
Comment (sguequierre) [inline docs/services/vision/segmentation.md:18]:
```suggestion
See our guide [Navigate with a Rover Base](/tutorials/services/navigate-with-rover-base/#nextsteps) for an example of automating obstacle avoidance with 3D object segmentation.
```

File: docs/services/vision/segmentation.md [hunk 8]
```diff
@@ -11,7 +11,11 @@ tags: ["vision", "computer vision", "CV", "services", "segmentation"]
 _Changed in [RDK v0.2.36 and API v0.1.118](/appendix/release-notes/#25-april-2023)_
 
 _3D Object Segmentation_ is the process of separating and returning a list of the identified "objects" from a 3D scene.
-The "objects" are a list of point clouds with associated metadata, like the label, the 3D bounding box, and center coordinates of the object.
+The "objects" are usually a list of point clouds with associated metadata, like the label, the 3D bounding box, and center coordinates of the object.
+They can also be single points, like with the `obstacles_distance` model.
```
Comment (sguequierre) [inline docs/services/vision/segmentation.md:15]:
```suggestion
```

File: docs/services/vision/segmentation.md [hunk 9]
```diff
@@ -11,7 +11,11 @@ tags: ["vision", "computer vision", "CV", "services", "segmentation"]
...
+
+3D object segmentation is useful for obstacle detection.
+See our guide [Navigate with a Rover Base](/tutorials/services/navigate-with-rover-base/#next-steps-automate-obstacle-detection) for an example of automating obstacle avoidance with 3D object segmentation.
```
Comment (sguequierre) [inline docs/services/vision/segmentation.md:18]:
```suggestion
See our guide [Navigate with a Rover Base](/tutorials/services/navigate-with-rover-base/#next-steps-automate-obstacle-detection) for an example of automating obstacle avoidance with 3D object segmentation for obstacle detection.
```

**Pull request merged**